webapp/src/main/resources/public/js/components/repositories/RepositoryRow.js

```diff
@@ -310,7 +320,7 @@ let RepositoryRow = React.createClass({
         let numberOfNeedsNeedsReview = this.getNumberOfNeedsReview(repoId);
         let numberOfOOSLA = this.getNumberOfOOSLA(repoId);
 
-        if (numberOfNeedsTranslation === 0 && numberOfNeedsNeedsReview === 0 && numberOfOOSLA === 0) {
+        if (this.getCheckSLA(repoId) && numberOfNeedsTranslation === 0 && numberOfNeedsNeedsReview === 0 && numberOfOOSLA === 0) {
```

Comment:
not sure to understand that why you added that condition for the "done" label. 

The project can be "done" regardless if the SLA check is on or not. I'd remove this test, no?

Reply:
make sense. Remove checkSLA for done Label.